scp.js

```diff
@@ -32,12 +41,14 @@ scp.send = function (options, cb) {
  * Grab a file from a remote host
  */
 scp.get = function (options, cb) {
+  var ssh_options = ["ControlMaster=no"]; //callback is not fired if ssh sessions are shared
```

Comment:
this is redundant given similar code in `buildSshOptions`, but also doesn't work because `buildSshOptions` expects ssh_options to be an object